src/plugins/intel_cpu/CMakeLists.txt

```diff
@@ -148,6 +148,7 @@ if(RISCV64_XUANTIE)
 else()
     set(ENABLE_SHL_FOR_CPU_DEFAULT OFF)
 endif()
+set(ENABLE_KLEIDIAI_FOR_CPU ON)
```

Comment:
please, use `ov_option`
In this case this option will be visible in cmake cache (CMakeCache.txt file) and cmake gui.